src/metabase/pulse/parameters.clj

```diff
@@ -15,19 +15,24 @@
   [_pulse dashboard]
   (:parameters dashboard))
 
+(defn param-val
```

Comment:
```suggestion
(defn- param-val
```
or maybe
```suggestion
(defn- param-val-or-default
```

Reply:
Thanks, went with the latter: 7070d65fd5491300c84f7242d382c708213b8c8f